api/events/task.proto

```diff
@@ -29,6 +29,9 @@ message TaskDelete {
 	uint32 pid = 2;
 	uint32 exit_status = 3;
 	google.protobuf.Timestamp exited_at = 4 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+	// id is the specific exec. By default if omitted will be `""` thus matches
+	// the init exec of the task matching `container_id`.
+	string id = 5;
```

Comment:
How about `exec_id` here? it can be more specific and consistent with existing API, like 

https://github.com/containerd/containerd/blob/a4f7b3758e7cd504ff99a1e4e1e123e8d52aa1db/api/events/task.proto#L61-L65

Reply:
I followed the `TaskExit` pattern which uses `container_id` `id`. I am not tied to it though so if you would like a rename I am happy. I do think `exec_id` is more clear. Seems the file has both patterns.

Reply:
alright. keep consistent with `TaskExit` and maybe we can change it all later 😄